static/js/main.js

```diff
@@ -1,48 +1,42 @@
-document.addEventListener("DOMContentLoaded", function(){
-  var toggle = document.getElementById("scheme-toggle");
-
-  var scheme = "light";
-  var savedScheme = localStorage.getItem("scheme");
-
-  var container = document.getElementsByTagName("html")[0];
-  var prefersDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
-
-  if (prefersDark) {
-    scheme = "dark";
-  }
+function toggleDarkMode() {
+    var theme = localStorage.getItem('scheme');
+    var toggle = document.getElementById('scheme-toggle');
+    var container = document.getElementsByTagName("html")[0];
+
+    if (theme == 'dark') {
```

Comment:
_Minor_: My Javascript is beyond rusty, but I think you want strict equality (`===`) when loading from local storage?

Reply:
Technically, yeah. Doesn't hurt either way in practice from what I've seen over the years, but I'll change it to be more "compliant."